faunadb/deprecated.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+import functools
+
+def deprecated(reason):
+  def decorator(old_func):
+
+    @functools.wraps(old_func)
+    def new_func(*args, **kvargs):
+      fmt = '{name}: {reason}'
+      warnings.simplefilter('always', DeprecationWarning)
```

Comment:
I don't know how I feel about modifying the warnings filter like this. I know these warnings are suppressed by default, but I don't like overriding a user's config ourselves.

Reply:
if we don't change this specific warning level the deprecate message won't show up to the users

Reply:
The deprecation message is an opt-in message. I don't like forcing it on outside the control of a user. Also, if a user already had enabled `DeprecationWarning`, this code would disable it after this has run. If we want to always print this message when this function is used (regardless of user settings), it might be better to use a logger warning instead. At the very least, that becomes opt-out.

Reply:
I tried to use loggers and I didn't like the result because it lost the context of the caller, ie doesn't show the correct file/line number of the deprecated function, only shows information referring to the `deprecated.py`.

ps: I'm not saying the we always should print deprecation information, I just said that by default the interpreter doesn't show it, so the user might need to enable the warnings information in the command line, but if you thing it's a desirable behavior (opt-in), I'm okay as well, so I can remove the filters.

Reply:
so, I'll remove the filters and let it to the developer to enable warning messages